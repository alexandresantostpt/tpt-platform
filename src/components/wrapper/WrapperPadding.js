import styled from 'styled-components'

const WrapperPadding = styled.div`
    margin-bottom: 4rem;
`

export default WrapperPadding
