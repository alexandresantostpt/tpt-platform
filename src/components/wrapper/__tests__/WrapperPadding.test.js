import React from 'react'
import renderer from 'react-test-renderer'

import WrapperPadding from '../WrapperPadding'

describe('Tests for WrapperPadding component', () => {
    it('Should render without crash', () => {
        const tree = renderer.create(<WrapperPadding />).toJSON()
        expect(tree).toMatchSnapshot()
    })
})
