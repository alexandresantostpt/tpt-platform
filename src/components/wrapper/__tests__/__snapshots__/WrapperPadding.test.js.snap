// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Tests for WrapperPadding component Should render without crash 1`] = `
<div
  className="WrapperPadding-ovjbyx-0 lcnoja"
/>
`;
